crates/sui-swarm/src/memory/swarm.rs

```diff
@@ -97,11 +97,19 @@ impl<R: ::rand::RngCore + ::rand::CryptoRng> SwarmBuilder<R> {
             .map(|config| (config.sui_address(), Node::new(config.to_owned())))
             .collect();
 
+        let mut fullnodes = HashMap::new();
+
+        if let Some(cnt) = fullnode_count {
+            (0..cnt).for_each(|_| {
+                let config = network_config.generate_fullnode_config();
```

Comment:
you don't need to fix this in this PR, but we will eventually need a way to configure nodes in tests. You could imagine eventually being able to do stuff like:

```
Swarm::builder()
   .add_fullnode(fullnode_config().random())
   .add_fullnode(
        fullnode_config()
           .default()
           .max_sync_concurrency(200)
   )
   .build()
```